### DISCUSSION TITLE
Importing Dense Vnet

### DISCUSSION THREAD

**Discussion by Eddowesselink** [Project-MONAI/MONAI#3916] [Q&A] [answered]
Dear contributors,

Are there any plans to import the Dense V-net architecture (comparable to the one in Niftynet) to Monai? We are aiming to compare this architecture to 3D Unet in future studies. 

Kind regards,
Eddo

**Comment by wyli** [comment, marked as answer]
(see also: https://github.com/Project-MONAI/MONAI/issues/90)
